### PR TITLE
Create Hidden-black.md

### PR DESCRIPTION
**Pull request by Hidden-black**
## Describe your project in 1-3 sentences

I am building a remote control Toy car with the help of Arduino, I also plan to use these parts for small projects like motion sensors and integrate it into the car (like when the car gets near a wall it turns) .

## Hack Club Slack handle
Shashwat Rai

## Checklist before submitting proposal:

- [+] I am a current high school student (or younger).
- [+] I have submitted an RSVP at <https://hackclub.com/winter>
- [+] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [+] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
@Hidden-black thank you for sharing these ideas hehe, they all sound awesome :) could you please elaborate on how you'll be going about each of your individual projects? and maybe focusing on 1-2 (like you mentioned the toy car) that you'd like to work on for the duration of the event?

please let me know if you have any questions :)) also, could you please factor in taxes/shipping? thank you!!

**Comment by Hidden-black** [comment]
am i suppose to type it in the markdown file? Also under which section? And how much explanation is needed

**Comment by bellesea** [comment]
@Hidden-black in response to your question, yes please add it in the markdown file under the ##plan section :) and 3-5 sentences each (or as much as you need) would be perfect!

File: Hidden-black.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+I first objective with this project will be to create a simple project like a motion sensor that detects when someone is coming near my room. I would also try to build diffrent things like a gas sensor before I get started with the main project to acquire the basic knowledge of Arduino.
+<br>
+The small toy car would help me with learning programing like controling all 4 motors seprately using motor driver
+
+### 1. Get The materials
+- Get everything required for this project along with other everyday thing (like cardboard)
+
+### 2. Build a frame for toy car
+- Plan the car in blender using simple shapes like cuboid and celender to get it proper placement and dimmensions of the components
+- Build the frame
```
Comment (bellesea) [inline Hidden-black.md:22]: @Hidden-black thanks for pushing the changes! how do you plan on building the frame?

also, is there a guide that you plan on referencing?

**Review by bellesea — CHANGES_REQUESTED** [review]

**Review by bellesea — APPROVED** [review]
@Hidden-black this is going to be super awesome!!! can't wait to see you work on this :) 

an awesome member of @hackclub/bank-ops will be reaching out with next steps!

**Review by MelSmith104 — COMMENTED** [review]
hi @Hidden-black ! i love that you're building a toy car! this project sounds like so much fun :) im dm'ing you on slack right now so that we can chat about next steps!

**Comment by bellesea** [comment]
heyy just checking, do you have a usb port on your computer that you can use to program the arduino?

**Comment by bellesea** [comment]
hey @Hidden-black :) you've been unresponsive w/ our team for a while so we're closing this PR, if you're interested in moving forward please reach out to your bank point of contact (or respond in this thread) within 48 hours. thank you!

**Pull request closed without merging**